inspirehep/modules/records/tasks.py

```diff
@@ -204,3 +210,38 @@ def actions():
         'success': success,
         'failures': [failure for failure in failures or []],
     }
+
+
+@shared_task(ignore_result=False, bind=True)
+def index_new_cited_records(self, pid_type, pid_value, db_version):
+    record = get_db_record(pid_type, pid_value)
+
+    if record.model.version_id < db_version:
+        backoff = 2 ** (self.request.retries + 1)
+        eta = datetime.datetime.now() + datetime.timedelta(seconds=backoff)
```

Comment:
Celery 4.2 has this builtin, but 4.1 doesn't seem to unfortunately.

Reply:
Doh